lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -228,14 +225,21 @@ class CategoryRenderer {
     wrapper.href = `#${category.id}`;
     wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
 
+    const numericScore = Number(category.score);
```

Comment:
should call out casting `null` to 0 here in a comment

Reply:
done